lib/reporters/html.js

```diff
@@ -84,6 +85,12 @@ function HTML(runner, options) {
     ctx = canvas.getContext('2d');
     ctx.scale(ratio, ratio);
     progress = new Progress();
+  } else {
+    // On some broswers, canvas might be unavailable for whatever reason.
+    // As such, we need a text version as a fallback
+    var progressTextFallback = fragment('<li class="progress-text">progress: <em>0</em>%</li>');
+    progressText = progressTextFallback.getElementsByTagName('em')[0];
+    items[0].replaceWith(progressTextFallback);
```

Comment:
[Refactor] It's not great to have to maintain two code paths for the same feature. Also, the [`<progress>` element is supported in all major browsers](https://caniuse.com/progress). Even IE 10! (which Mocha no longer supports)

Proposal: let's remove the canvas altogether, and switch to a full HTML/CSS `<progress>`?

Reply:
Should I also be removing progress.js?

Reply:
If the code is no longer used, then 🔪 let's remove it!